src/main/java/core/basesyntax/database/DataBase.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.database;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DataBase {
+    private static final Map<String, Integer> db = new HashMap<>();
+    private static final String DATA_FILE_PATH = "./src/main"
+            + "/java/core/basesyntax/database/beginningData";
+    private static final String REPORT_FILE_PATH = "./src/main"
+            + "/java/core/basesyntax/database/report";

```

Comment:
not correct
https://github.com/mate-academy/jv-fruit-shop/blob/master/checklist.md#place-the-input-and-output-files-into-the-srcmainresources-folder

Reply:
look up folder strucutre of a manen project 
src/main/java, src/main/resources, src/test/java, src/test/resources